mava/types.py

```diff
@@ -87,6 +86,18 @@ class PPOTransition(NamedTuple):
     info: Dict
 
 
+class SebulbaTransition(NamedTuple):
```

Comment:
We should try keep this to the same transition. If we're going to make a new transition for each system then those types should go in their own files